amr-wind/physics/multiphase/MultiPhase.cpp

```diff
@@ -13,59 +15,127 @@ MultiPhase::MultiPhase(CFDSim& sim)
-        m_interface_tracking_method = amr_wind::InterfaceTrackingMethod::LS;
+        m_interface_capturing_method = amr_wind::InterfaceCapturingMethod::LS;
         auto& levelset_eqn =
             sim.pde_manager().register_transport_pde("Levelset");
         m_levelset = &(levelset_eqn.fields().field);
     } else {
-        amrex::Print() << "Please select an interface tracking model between "
+        amrex::Print() << "Please select an interface capturing model between "
                           "VOF and Levelset: defaultin to VOF "
                        << std::endl;
-        m_interface_tracking_method = amr_wind::InterfaceTrackingMethod::VOF;
+        m_interface_capturing_method = amr_wind::InterfaceCapturingMethod::VOF;
         auto& vof_eqn = sim.pde_manager().register_transport_pde("VOF");
         m_vof = &(vof_eqn.fields().field);
         // Create levelset as a auxilliary field only !
         m_levelset = &(m_sim.repo().get_field("levelset"));
+        const amrex::Real levelset_default = 0.0;
+        BCScalar bc_ls(*m_levelset);
+        bc_ls(levelset_default);
     }
 }
 
-InterfaceTrackingMethod MultiPhase::interface_tracking_method()
+InterfaceCapturingMethod MultiPhase::interface_capturing_method()
 {
-    return m_interface_tracking_method;
+    return m_interface_capturing_method;
 }
 
 void MultiPhase::post_init_actions()
 {
 
-    if (m_interface_tracking_method == InterfaceTrackingMethod::VOF) {
+    if (m_interface_capturing_method == InterfaceCapturingMethod::VOF) {
         levelset2vof();
         set_density_via_vof();
-    } else if (m_interface_tracking_method == InterfaceTrackingMethod::LS) {
+    } else if (m_interface_capturing_method == InterfaceCapturingMethod::LS) {
         set_density_via_levelset();
     }
     m_density.fillpatch(m_sim.time().current_time());
 }
 
 void MultiPhase::post_advance_work()
 {
-    if (m_interface_tracking_method == InterfaceTrackingMethod::VOF) {
+    if (m_interface_capturing_method == InterfaceCapturingMethod::VOF) {
         set_density_via_vof();
-    } else if (m_interface_tracking_method == InterfaceTrackingMethod::LS) {
+    } else if (m_interface_capturing_method == InterfaceCapturingMethod::LS) {
         set_density_via_levelset();
     }
     m_density.fillpatch(m_sim.time().current_time());
+
+    // Compute the total volume fraction
+    if (m_interface_capturing_method == InterfaceCapturingMethod::VOF) {
+        m_total_volfrac = volume_fraction_sum();
+        const auto& geom = m_sim.mesh().Geom();
+        const amrex::Real total_vol = geom[0].ProbDomain().volume();
+        amrex::Print() << "Volume of Fluid diagnostics:" << std::endl;
+        amrex::Print() << "   Water Volume Fractions Sum : " << m_total_volfrac
+                       << std::endl;
+        amrex::Print() << "   Air Volume Fractions Sum : "
+                       << total_vol - m_total_volfrac << std::endl;
+        amrex::Print() << " " << std::endl;
```

Comment:
Protect print stuff with `verbose` or remove it.

Reply:
```suggestion
    // Compute the print the total volume fraction
    if (m_interface_capturing_method == InterfaceCapturingMethod::VOF &&
        m_verbose > 0) {
        m_total_volfrac = volume_fraction_sum();
        const auto& geom = m_sim.mesh().Geom();
        const amrex::Real total_vol = geom[0].ProbDomain().volume();
        amrex::Print() << "Volume of Fluid diagnostics:" << std::endl;
        amrex::Print() << "   Water Volume Fractions Sum : " << m_total_volfrac
                       << std::endl;
        amrex::Print() << "   Air Volume Fractions Sum : "
                       << total_vol - m_total_volfrac << std::endl;
        amrex::Print() << " " << std::endl;
    }
```